HousingSearchApi/V1/Infrastructure/Factories/TenureQueryGenerator.cs

```diff
@@ -30,15 +34,18 @@ public QueryContainer Create<TRequest>(TRequest request, QueryContainerDescripto
             if (tenureListRequest.SearchText != null && tenureListRequest.SearchText.Length > 0)
             {
 
-                return _queryBuilder
-                .WithWildstarQuery(tenureListRequest.SearchText, new List<string>
-                {
-                    "paymentReference",
-                    "tenuredAsset.fullAddress^3",
-                    "householdMembers",
-                    "householdMembers.fullName^3"
-                })
-                .Build(q);
+                return _queryFilterBuilder
```

Comment:
This executes only when you have a search text.
Does your case require a search text?

Reply:
You are correct that we won't always have search text however the current endpoint doesn't allow empty search text and it requires tweaking the endpoint contracts by reducing the validation whilst it is unclear if the endpoint can handle the reduced validation. This code block will be triggered as we send empty quotes and works as intended.

Reply:
Yup, that makes sense.

Based on this. It sounds like these changes would be better off in a separate endpoint, but I'm guessing this is a deliverable too large given it would involve a lot work dealing with these wrappers.